TypeCobol.LanguageServer/Workspace.cs

```diff
@@ -717,7 +717,7 @@ private void RefreshCustomSymbols()
                     LoadingIssueEvent(null, new LoadingIssueEvent() { Message = "An error occured while trying to load Intrinsics or Dependencies files." }); //Send notification to client
 
                 LoggingSystem.LogException(exception);
```

Comment:
Is it possible to pass CompilationProject.CompilationOptions here so the logger can also log the UserAgent?
```suggestion
                LoggingSystem.LogException(exception, CompilationProject.CompilationOptions);
```

Reply:
As discussed, user-agent is always logged in every trace as part of the command-line.